backend/src/main/java/codezap/auth/controller/AuthController.java

```diff
@@ -32,20 +40,28 @@ public ResponseEntity<LoginResponse> login(
     ) {
         LoginMember loginMember = authService.login(loginRequest);
         Credential credential = credentialProvider.createCredential(loginMember);
-        credentialManager.setCredential(httpServletResponse, credential);
+        credentialManagers.forEach(
+                credentialManager -> credentialManager.setCredential(httpServletResponse, credential)
+        );
```

Comment:
혹시 개행 안 하는 건 어떤가요...? 120줄을 넘지 않고, 안 하는 게 더 깔금해보여요

```suggestion
        credentialManagers.forEach(credentialManager -> credentialManager.setCredential(httpServletResponse, credential));
```